File: CS/dxSample/Form1.cs
using System;
using System.Collections.Generic;
using System.Linq;
using DevExpress.XtraReports.UI;

namespace dxSample {
    public partial class Form1 : DevExpress.XtraBars.Ribbon.RibbonForm {
        public Form1() {
            InitializeComponent();
        }
        string reportDefinitionFilePath = "..\\..\\Reports\\XtraReport1.repx";
        private void Form1_Load(object sender, EventArgs e) {
            reportDesigner1.OpenReport(reportDefinitionFilePath);
        }
    }
}

File: VB/dxSample/Form1.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports DevExpress.XtraReports.UI

Namespace dxSample
    Partial Public Class Form1
        Inherits DevExpress.XtraBars.Ribbon.RibbonForm

        Public Sub New()
            InitializeComponent()
        End Sub
        Private reportDefinitionFilePath As String = "..\..\Reports\XtraReport1.repx"
        Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
            reportDesigner1.OpenReport(reportDefinitionFilePath)
        End Sub
    End Class
End Namespace
